com.cubrid.cubridmigration.core/src/com/cubrid/cubridmigration/core/engine/importer/impl/OfflineImporter.java

```diff
@@ -282,14 +284,25 @@ public int writeData(
 
                     int index = 0;
                     for (String val : res) {
-                        sheet.addCell(new jxl.write.Label(index++, total, val));
+                        if (val.length() > 32767) {
```

Comment:
How about using a fixed constant value as a global variable?
ex) `private static final int MAX_RECORD_LENGTH`

Reply:
Thanks for the advice, fixed it. (MAX_EXCEL_CELL_LENGTH)